core/src/test/scala/unit/kafka/server/FetchRequestDownConversionConfigTest.scala

```diff
@@ -163,7 +164,9 @@ class FetchRequestDownConversionConfigTest extends BaseRequestTest {
   }
 
   def testV1Fetch(isFollowerFetch: Boolean): Unit = {
+    val fetchMessageConversionsTimeMsMetricName = s"$MessageConversionsTimeMs,request=Fetch"
```

Comment:
nit

Is it possible to generate this string with some constants? e.g. we can use `RequestMetrics.MessageConversionsTimeMs`

Reply:
Yes, we already used the constants from `RequestMetrics.MessageConversionsTimeMs`. It's just the fetch/produce request is another tag name that needed to be filtered out. Thanks.